aiidalab_widgets_base/process.py

```diff
@@ -545,10 +542,10 @@ class ProcessListWidget(ipw.VBox):
 
     past_days (int): Sumulations that were submitted in the last `past_days`.
 
-    incoming_node (int, str, Node): Trait that takes node id or uuid and returns the node that must
+    incoming_node (tr): Trait that takes node uuid that must
```

Comment:
Any reason why not keep support of passing `id` instead of `uuid`?

Reply:
You mean `pk`? Do you have any use case for where you what to use primary key, since I think you can use UUID to include all the cases for `pk`?